doc/user/olm-catalog/operator-bundles.md

```diff
@@ -0,0 +1,23 @@
+# Operator Bundles
+
+This document gives an overview of using `operator-sdk` to work with Operator [bundles][registry-bundle], namely on-disk bundle directories and creating bundle [images/metadata][registry-bundle-image].
+
+## Commands
+
+The following `operator-sdk` subcommands create or interact with Operator on-disk bundles and bundle images:
+
+* `generate csv`: creates a new or updates an existing CSV in a semantically versioned bundle directory, creates a package manifest if it does not exist, and optionally copies your CRDs to the versioned bundle directory. Read more about this command [here][sdk-generate-csv].
+* `bundle create`: creates an Operator bundle image from manifests on disk, or writes bundle image metadata to disk. This subcommand has corresponding functionality to `opm alpha bundle build`, and is stable. Output and underlying behavior between these commands is the same, except nothing is written to disk unless `--generate-only` is set (unset by default). Refer to [`opm alpha bundle build` docs][registry-opm-build] for more information. CLI differences between these commands:
+    * `opm alpha bundle build --default-channel=<channel-name>` equates to `operator-sdk bundle create --default=<channel-name>`.
```

Comment:
The opm commands are not part of SDK. Am I right? 
Could we clarify where they came from?